components/textbox.tsx

```diff
@@ -2,29 +2,27 @@ import * as React from 'react';
 import styles from "../styles/Textbox.module.css";
 import Pointsbox from "./pointsbox";
 import {useState} from "react";
+import {AssessmentType} from "../types/Types";
+
+
 
 
-export interface Option {
-    value: number;
-    label: string;
-}
 interface textboxprop {
-    text: string |undefined;
-    maxPoints: number;
+    assessment: AssessmentType;
+    setAssessment: (assessment: AssessmentType, newScore: number) => void;
 }
 
 const Textbox: React.FC<textboxprop> = (props: textboxprop) => {
-    const [selectedOption, setSelectedOption] = useState<Option | null | undefined>();
+    const {assessment, setAssessment} = props;
     return (
         <div className={styles.card}>
             <div className={styles.alignTitlePoints}>
                 <Pointsbox
-                    maxPoints={props.maxPoints}
-                    selectedOption={selectedOption}
-                    setSelectedOption={setSelectedOption}
+                    assessment={assessment}
+                    setAssessment={setAssessment}
                 />
             </div>
-            {props.text}
+            {props.assessment.answer}
```

Comment:
do we need props here?

Reply:
Uncertain... But I would rather argue that we can delete `line 16` and use `props.` on line 21 and 22 as well.
Do you remember why we have `line 16`?
> `const {assessment, setAssessment} = props;`